run/idp-sql/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "idp-sql",
+  "description": "Identity Platform and Cloud SQL integrations sample for Cloud Run.",
+  "version": "0.0.1",
+  "private": true,
+  "license": "Apache-2.0",
+  "author": "Google LLC",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/GoogleCloudPlatform/nodejs-docs-samples.git"
+  },
+  "engines": {
+    "node": ">=10.0.0"
```

Comment:
fs.promises was introduced during the v10 lifecycle. Not sure how specific we want to get here, but I think claiming not to support earlier than the Dockerfile is fair.

Reply:
done.